lms/djangoapps/course_blocks/transformers/hidden_content.py

```diff
@@ -27,7 +27,9 @@ class HiddenContentTransformer(FilteringTransformerMixin, BlockStructureTransfor
     """
     VERSION = 1
```

Comment:
You need to increment the version number on the transformer otherwise you'll get an exception in stage/production since you'll be working off of previously collected data.